src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -70,5 +102,21 @@ public RobotMap() {
         HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.xboxController);
 		HumanInput.Operator.joystick = new CustomJoystick(Port.HumanInput.joystick);
 
+        CANTalonFX flywheelATalon = new CANTalonFX(Port.CANMotor.FLYWHEEL_MOTOR_A);
+        // flywheelATalon.setInverted(false);
+        CANTalonFX flywheelBTalon = new CANTalonFX(Port.CANMotor.FLYWHEEL_MOTOR_B);
```

Comment:
Make sure that one of these is inverted so that the flywheel can actually shoot.